runtime/compiler/optimizer/EscapeAnalysis.cpp

```diff
@@ -1945,7 +1949,102 @@ void TR_EscapeAnalysis::checkDefsAndUses()
       }
    }
 
+void TR_EscapeAnalysis::printUsesThroughAternary(void)
+   {
+   traceMsg(comp(), "\nNodes used through aternary operations\n");
+
+   for (auto mi = _nodeUsesThroughAternary->begin(); mi != _nodeUsesThroughAternary->end(); mi++)
+      {
+      TR::Node *key = mi->first;
+      int32_t nodeIdx = key->getGlobalIndex();
+
+      traceMsg(comp(), "   node [%p] n%dn is used by {", key, nodeIdx);
+
+      TR_ArrayIterator<TR::Node> ai(mi->second);
+      bool first = true;
+
+      for (TR::Node* aternaryNode = ai.getFirst(); aternaryNode; aternaryNode = ai.getNext())
+         {
+         traceMsg(comp(), "%s[%p] n%dn", (first ? "" : ", "), aternaryNode,
+                  aternaryNode->getGlobalIndex());
+         first = false;
+         }
+
+      traceMsg(comp(), "}\n");
+      }
+   }
 
+void TR_EscapeAnalysis::gatherUsesThroughAternary(void)
+   {
+   TR::NodeChecklist visited(comp());
+   TR::TreeTop *tt = comp()->getStartTree();
+
+   for (; tt; tt = tt->getNextTreeTop())
+      {
+      TR::Node *node = tt->getNode();
+      gatherUsesThroughAternaryImpl(node, visited);
+      }
+
+   if (trace() && _nodeUsesThroughAternary)
+      {
+      printUsesThroughAternary();
```

Comment:
probably a good idea to say that there were no aternary opcodes found or something rather than being silent.

Reply:
Added a message in the case where no `aternary` opcodes were found in commit https://github.com/eclipse/openj9/pull/5699/commits/f822f8ae1f3604b2a5c3a855d3476f2390a9c4e0